pynecone/utils.py

```diff
@@ -1228,3 +1230,46 @@ def get_redis() -> Optional[Redis]:
 
 # Store this here for performance.
 StateBases = get_base_class(StateVar)
+
+# Helper functions for telemetry.
+def get_os() -> str:
+    """Get the operating system.
+
+    Returns:
+        The operating system.
+    """
+    return platform.system()
+
+
+def get_python_version() -> str:
+    """Get the Python version.
+
+    Returns:
+        The Python version.
+    """
+    return platform.python_version()
+
+
+def get_cpu_count() -> int:
+    """Get the number of CPUs.
+
+    Returns:
+        The number of CPUs.
+    """
+    return multiprocessing.cpu_count()
+
+
+def add_telemetry() -> Dict:
+    """Add telemetry to an event.
+
+    Returns:
+        The payload with telemetry added.
+    """
+    payload = {}
```

Comment:
I think we can create a class `Telemetry(pc.Base)` with these fields to make it strongly typed - then we can call the `.json()` method to get this dict.

We can make a new file `telemetry.py` and define the class and this method there